appium/tests/specs/mock/setup/CheckRefreshKeyFromEkmKeyDeletionAndRevokedKey.spec.ts

```diff
@@ -13,10 +13,16 @@ describe('SETUP: ', () => {
 
   it('will not update a revoked private key with valid one and delete local key if it\'s removed from EKM and not revoked one', async () => {
 
-    const mockApi = MockApi.e2eMock;
```

Comment:
Could call it `mockWithDefaultEnterpriseConfig`

Reply:
Ok, will rename it